CGBEmu/Origem.cpp

```diff
@@ -1,10 +0,0 @@
-#include <iostream>
```

Comment:
There can only be one main function in a project. (i.e the main function is the entry point of the application).
You got the weird compiler issues because you had added this file that contained an additional main function.
To solve this issue I removed the file completely from the project.